lighthouse-core/report/v2/report-renderer.js

```diff
@@ -215,12 +243,15 @@ window.ReportRenderer = class ReportRenderer {
     const element = this._createElement('div', 'lighthouse-audit');
     let title = audit.result.description;
     if (audit.result.displayValue) {
-      title += ': ' + audit.result.displayValue;
+      title += `:  ${audit.result.displayValue}`;
     }
-    element.appendChild(this._renderScore(audit.score, title, audit.result.helpText));
-    if (audit.result.details) {
-      element.appendChild(this._renderDetails(audit.result.details));
+    if (audit.result.optimalValue) {
+      title += ` (target: ${audit.result.optimalValue})`;
     }
+
+    element.appendChild(
+        this._renderScore(audit.result.score, title, audit.result.helpText, audit));
```

Comment:
Yeah moved discussion to https://github.com/GoogleChrome/lighthouse/pull/1963#discussion_r109763756 :)